### DISCUSSION TITLE
error .. file not found

### DISCUSSION THREAD

**Discussion by dperezch** [quasarframework/app-extension-qpdfviewer#116] [Q&A]
hello, the plugin works perfect in html, but when i use the android emulator the pdf is not seen, i try to change the path by removing and adding '/' or using a full url but the pdf file is not seen, when i  build, the next error appears in the app: web page not available, file:///pdfjs/web/viewer.html..... could not be loaded because:
ERR_FILE_NOT_FOUND

what can i do?
![Screenshot_20220403-143419_Energiza App](https://user-images.githubusercontent.com/75341308/161442955-04f5d2b6-dd57-4e83-8728-4a5a0b6bf7d8.jpg)
![emulator](https://user-images.githubusercontent.com/75341308/161442956-73b384b3-6a53-4913-931c-08adf8326251.png)

**Comment by hawkeye64** [comment]
Sorry, but I don't have experience enough with Android to help here. :(

**Reply by hawkeye64** [reply]
Hopefully, someone from the community can help.

**Comment by nucle** [comment]
Hi @davidperez-curso ,
can u publish a part of your code which is running in the emulator?
Then i can take a look.

BR,
nucle
